thirdparty/optionparser/optionparser.h

```diff
@@ -1556,9 +1556,9 @@ inline bool Parser::workhorse(bool gnu, const Descriptor usage[], int numargs, c
 
     do // loop over short options in group, for long options the body is executed only once
     {
-      int idx(0);
+      int idx;
 
-      const char* optarg = nullptr;
+      const char* optarg;
```

Comment:
Windows CI is complaining about this variables being uninitialized. I think we should revert these two changes.